templates/includes/sidebar.html

```diff
@@ -0,0 +1,14 @@
+<ul class="sidebar">
+  <li>
+    <a href="#">Home page</a>
+  </li>
+  <li>
+    <a href="#">Manufacturers</a>
+  </li>
+  <li>
+    <a href="#">Cars</a>
+  </li>
+  <li>
+    <a href="#">Drivers</a>
```

Comment:
The href attributes in the anchor tags are currently set to '#'. Consider updating these to point to the actual URLs within your application to make the sidebar functional.